build_system/src/build.rs

```diff
@@ -204,11 +209,20 @@ fn build_codegen(args: &mut BuildArg) -> Result<(), String> {
     // We voluntarily ignore the error.
     let _ = fs::remove_dir_all("target/out");
     let gccjit_target = "target/out/gccjit";
-    create_dir(gccjit_target)?;
-
-    println!("[BUILD] sysroot");
-    build_sysroot(&env, &args.config_info)?;
+    fs::create_dir_all(gccjit_target).map_err(|error| {
+        format!(
+            "Failed to create directory `{}`: {:?}",
+            gccjit_target, error
+        )
+    })?;
+   
```

Comment:
Perhas you could try print the field to see if we actually reach this line before the `if`:
```
println!("Build sysroot: {}", args.sysroot);
```

Reply:
Yaah, i will try this

Reply:
It is not even printed, so we don't even reach this code.
Do you have any idea what in this PR could have the effect of skipping this code?

Reply:
I suspect the build command

Reply:
so far, this is the only place, we added the sysroot flag   
```
- name: Build
      run: |
        ./y.sh prepare --only-libcore
        ./y.sh build --sysroot
        cargo test
```

in the ci.yml file